public/components/trace_analytics/components/traces/trace_view_helpers.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import moment from 'moment';
+
+export function normalizePayload(parsed: any): any[] {
+  if (Array.isArray(parsed)) {
+    return parsed;
+  }
+  if (parsed.hits && Array.isArray(parsed.hits.hits)) {
+    return parsed.hits.hits;
+  }
+  return [];
+}
```

Comment:
Why do we need this function to `normalizePayload`? Since we are using just 1 request for all trace related details shouldn't we normalize this at source where we receive the request?

Reply:
Moved to traces_request_handler
and used in handlePayloadRequest
```
const normalizedData = normalizePayload(response);
setPayloadData(JSON.stringify(normalizedData, null, 2));
```